source/simulator/assemblers/newton_stokes.cc

```diff
@@ -229,7 +251,7 @@ namespace aspect
                            // speaking, we probably ought to also
                            // consider the derivatives deta/deps
                            // here, but we leave this as a TODO
-                           one_over_eta
+                           (one_over_eta - dilation_newton_factor)
```

Comment:
yes, here you already precomputated `dilation_newton_factor` so this can stay as it is.